File: src/BukkitPE/level/particle/CriticalParticle.php
<?php
namespace BukkitPE\level\particle;

use BukkitPE\math\Vector3;

class CriticalParticle extends GenericParticle{
	public function __construct(Vector3 $pos, $scale = 2){
		parent::__construct($pos, Particle::TYPE_CRITICAL, $scale);
	}
}

File: src/main/java/net/BukkitPE/level/particle/CriticalParticle.java
package net.BukkitPE.level.particle;

import net.BukkitPE.math.Vector3;

/**
 * Created on 2015/11/21 by xtypr.
 * Package net.BukkitPE.level.particle in project BukkitPE .
 */
public class CriticalParticle extends GenericParticle {
    public CriticalParticle(Vector3 pos) {
        this(pos, 2);
    }

    public CriticalParticle(Vector3 pos, int scale) {
        super(pos, Particle.TYPE_CRITICAL, scale);
    }
}
